pages/apis/graphql/graphql_cookbook.md

```diff
@@ -540,7 +540,80 @@ mutation {
   }
 }
 ```
+## Clusters
 
+A collection of common tasks with clusters using the GraphQL API
+
+### Create a queue
+
+Creates a new queue in a cluster
```

Comment:
Style: Make this a full sentence and use punctuation to link it to the example.

```suggestion
Create a new queue in a cluster using the following call:
```

^ This applies to each entry.